### PR TITLE
PACA-720: fixing RDA API phase parsing bug

### PR DESCRIPTION
**Pull request by ericbuckley**
**JIRA Ticket:**
[PACA-720](https://jira.cms.gov/browse/PACA-720)

**User Story or Bug Summary:**
Fix the bug that has stopped RDA ETL data ingestion

---

### What Does This PR Do?
We are incorrectly parsing the phase information out of the message source attribute. We should be looking for "P\d" and not "p\d". This is causing errors since all the phases begin with a capital P.

### What Should Reviewers Watch For?
* Verify all PR security questions and checklists have been completed and addressed.

### What Security Implications Does This PR Have?

Submitters should complete the following questionnaire:

* If the answer to any of the questions below is **Yes**, then **you must** supply a link to the associated Security Impact Assessment (SIA), security checklist, or other similar document in Confluence here: **N/A**

    * Does this PR add any new software dependencies? 
      * [ ] Yes
      * [x] No
    * Does this PR modify or invalidate any of our security controls?
      * [ ] Yes
      * [x] No
    * Does this PR store or transmit data that was not stored or transmitted before?
      * [ ] Yes
      * [x] No

* If the answer to any of the questions below is **Yes**, then please add @StewGoin as a reviewer, and note that this PR **should not be merged** unless/until he also approves it.
    * Do you think this PR requires additional review of its security implications for other reasons?
      * [ ] Yes
      * [x] No

### What Needs to Be Merged and Deployed Before this PR?

This PR cannot be either merged or deployed until the following prerequisite changes have been fully deployed:

* N/A

### Submitter Checklist

I have gone through and verified that...:

* [x] I have named this PR and branch so they are [automatically linked](https://confluence.atlassian.com/adminjiracloud/integrating-with-development-tools-776636216.html) to the (most) relevant Jira issue. Ie: `BFD-123: Adds foo`
* [x] This PR is reasonably limited in scope, to help ensure that:
    1. It doesn't unnecessarily tie a bunch of disparate features, fixes, refactorings, etc. together.
    2. There isn't too much of a burden on reviewers.
    3. Any problems it causes have a small "blast radius".
    4. It'll be easier to rollback if that becomes necessary.
* [x] This PR includes any required documentation changes, including `README` updates and changelog / release notes entries.
* [x] All new and modified code is appropriately commented, such that the what and why of its design would be reasonably clear to engineers, preferably ones unfamiliar with the project.
* [x] All tech debt and/or shortcomings introduced by this PR are detailed in `TODO` and/or `FIXME` comments, which include a JIRA ticket ID for any items that require urgent attention.
* [x] Reviews are requested from both:
    * At least two other engineers on this project, at least one of whom is a senior engineer or owns the relevant component(s) here.
    * Any relevant engineers on other projects (e.g. DC GEO, BB2, etc.).
* [x] Any deviations from the other policies in the [DASG Engineering Standards](https://github.com/CMSgov/cms-oeda-dasg/blob/master/policies/engineering_standards.md) are specifically called out in this PR, above.
    * Please review the standards every few months to ensure you're familiar with them.

### CODE REVIEW

File: apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/source/AbstractClaimTransformer.java [hunk 1]
```diff
@@ -11,9 +11,9 @@ public abstract class AbstractClaimTransformer {
   /** Maps strings sent by RDA to simple numeric values for DB storage */
   private static final Map<String, Short> PHASE_TO_SHORT =
       Map.of(
-          "p1", (short) 1,
-          "p2", (short) 2,
-          "p3", (short) 3);
+          "P1", (short) 1,
+          "P2", (short) 2,
+          "P3", (short) 3);
```
[the remark this answers is not in the document]
Comment (brianburton) [reply apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/source/AbstractClaimTransformer.java:16]: This change has been implemented.

**Review by lsmitchell — APPROVED** [review]

File: apps/bfd-pipeline/bfd-pipeline-rda-bridge/src/main/java/gov/cms/bfd/pipeline/bridge/etl/FissTransformer.java [hunk 2]
```diff
@@ -202,7 +202,7 @@ FissClaimChange transformNewClaim(
             RecordSource.newBuilder()
```
Comment (mjburling) [inline apps/bfd-pipeline/bfd-pipeline-rda-bridge/src/main/java/gov/cms/bfd/pipeline/bridge/etl/FissTransformer.java:202]: There is nothing wrong with this line, but this PR looks like it's _just_ about ready to merge. Just holding it until 1292 lands.

Comment (mjburling) [inline apps/bfd-pipeline/bfd-pipeline-rda-bridge/src/main/java/gov/cms/bfd/pipeline/bridge/etl/FissTransformer.java:202]: There is nothing wrong with this line, but this PR looks like it's _just_ about ready to merge. Just holding it until 1292 lands.

Comment (ericbuckley) [reply apps/bfd-pipeline/bfd-pipeline-rda-bridge/src/main/java/gov/cms/bfd/pipeline/bridge/etl/FissTransformer.java:202]: @mjburling did you mean PR #1294?  This PR is 1292.

Comment (mjburling) [reply apps/bfd-pipeline/bfd-pipeline-rda-bridge/src/main/java/gov/cms/bfd/pipeline/bridge/etl/FissTransformer.java:202]: > @mjburling did you mean PR #1294? This PR is 1292.

Oh, dear. Yes, indeed. Now that this is through, I think we're all set. Sorry for the delay, @ericbuckley.

**Pull request merged**